### PR TITLE
Updating PauseSessionResponseResource object to exclude extraneous se…

### PR DESCRIPTION
**Pull request by burzum619**
…ssion information. TODO - set affected examIds once exam service piece is finished.

### CODE REVIEW

**Review by jeffjohnson9046 — APPROVED** [review]
looks good to me.

File: service/src/main/java/tds/session/web/resources/PauseSessionResponseResource.java [hunk 1]
```diff
@@ -14,24 +16,41 @@
...
         this.add(linkTo(
                 methodOn(SessionController.class)
-                .findSessionById(pauseSessionResponse.getSession().getId())
+                .findSessionById(session.getId())
                 ).withRel("session"));
     }
 
-    public PauseSessionResponse getPauseSessionResponse() {
-        return pauseSessionResponse;
+    public String getStatus() {
```
Comment (millerts) [inline service/src/main/java/tds/session/web/resources/PauseSessionResponseResource.java:39]: @jeffjohnson9046 - one thing I didn't think about when you suggested this is how would a client unmarshal this?  We try to keep our objects that we want clients to use in the client jar.

**Review by millerts — APPROVED** [review]
Changes look good

File: service/src/main/java/tds/session/web/resources/PauseSessionResponseResource.java (hunk 1 above)
Comment (jeffjohnson9046) [reply service/src/main/java/tds/session/web/resources/PauseSessionResponseResource.java:39]: Ugh - you're right.  The `PauseSessionResponse` in the `client` should probably be updated to match what the resource looks like.

Comment (burzum619) [reply service/src/main/java/tds/session/web/resources/PauseSessionResponseResource.java:39]: Makes sense to me, I will update PauseSessionResponse accordingly.

**Pull request merged**